crates/cli/src/lib.rs

```diff
@@ -574,8 +586,8 @@ impl Rustac {
 
     /// Returns the set or inferred input format.
     pub fn input_format(&self, href: Option<&str>) -> Format {
-        if let Some(input_format) = self.input_format {
-            input_format
+        if let Some(input_format) = &self.input_format {
+            *input_format
```

Comment:
Leftover, I think this can stay unchanged:

```suggestion
        if let Some(input_format) = self.input_format {
            input_format
```